app/views/publishers/notifications/index.html.slim

```diff
@@ -15,7 +15,7 @@
             .govuk-body.float-right
               = pagy_stats(@pagy, type: "notification")
 
-        = render @notifications
+        = render partial: "notification", collection: @notifications, as: :notification
```

Comment:
Due to notifications now being called noticed_notifications we need to explicitly set this collection partial.